lib/jbuilder/railtie.rb

```diff
@@ -16,11 +16,9 @@ module ApiRendering
           end
         end
 
-        ActiveSupport.on_load :action_controller do
-          if self == ActionController::API
```

Comment:
I think I could have also changed it to be `self.name == "ActionController::API"` to avoid breaking any Rails version
Or something like:
```ruby
# not sure about the logic behind constant_loaded? method
constant_loaded?("ActionController::API") && (self == ActionController::API)
```

But I find relying on the `:action_controller_api` the most elegant and correct solution loading-wise